src/odemis/acq/stream/_static.py

```diff
@@ -1053,6 +1053,11 @@ def update(self, raw):
 
 
 class FastEMOverviewStream(StaticSEMStream):
-    # For now just a StaticStream with a different name, so the canvas can automatically select the right
-    # blending option ("blend screen" on non-overlapping positions = simple pasting without blending)
-    pass
+
+    def _find_metadata(self, md):
+        useful_md = super()._find_metadata(md)
+        if 'merge_ratio' in md:
+            useful_md['merge_ratio'] = md['merge_ratio']
+        if 'blend_mode' in md:
+            useful_md['blend_mode'] = md['blend_mode']
+        return useful_md
```

Comment:
There is already a mechanism in LiveStream that could be used for here too: forcemd. It allows to apply some extra metadata on top of the metadata in the raw data. Please use this mechanism here too. Either just in this class, or in StaticStream, as it could be useful in other cases too.